code-samples/miscellaneous/parsing-mention-arguments/12/index.js

```diff
@@ -28,7 +28,7 @@ function getUserFromMentionRegEx(mention) {
 	return client.users.cache.get(id);
 }
 
-client.on('message', message => {
+client.on('message', async message => {
```

Comment:
```suggestion
client.on('message', message => {
```

Reply:
The `async` is necessary for `await guild.members.ban(...)` to work.